functions/that-api-webhooks/hooks/receive/auth0SignUp.js

```diff
@@ -0,0 +1,136 @@
+const Sentry = require('@sentry/node');
+const debug = require('debug');
+const ac = require('../send/activeCampaign');
+
+const dlog = debug('that:api:webhooks:auth0SignUp');
+
+module.exports = (req, res) => {
```

Comment:
do you really want to go with this, why not use express to export like we do with the others?

Reply:
What do you mean use express to export? I am exporting a function here used by express use()

Reply:
Done